ddtrace/settings/config.py

```diff
@@ -808,12 +808,17 @@ def _format_tags(self, tags: List[Union[str, Dict]]) -> Dict[str, str]:
     def enable_remote_configuration(self):
         # type: () -> None
         """Enable fetching configuration from Datadog."""
+        from ddtrace.internal.flare.flare import Flare
+        from ddtrace.internal.flare.handler import _handle_tracer_flare
+        from ddtrace.internal.flare.handler import _tracerFlarePubSub
```

Comment:
Had to move these to be in-line to fix `ModuleWatchdog is already installed` errors.